src/cases/NgRestTestCase.php

```diff
@@ -300,5 +351,77 @@ public function beforeTearDown()
         $this->userGroupFixture->cleanup();
         $this->userOnlineFixture->cleanup();
         $this->ngrestLogFixture->cleanup();
+        $this->app->db->createCommand()->dropTable('admin_auth')->execute();
+        $this->app->db->createCommand()->dropTable('admin_group_auth')->execute();
+        $this->app->db->createCommand()->dropTable('admin_user_group')->execute();
+    }
+
+    protected function removeApiPermissions()
+    {
+        $this->app->db->createCommand()->delete('admin_auth', ['id' => self::ID_AUTH_API])->execute();
+    }
+
+    protected function resetApiPermissions($create = false, $update = false, $delete = false)
```

Comment:
protected methods are available trough inheritance, therefore php doc is required including a @since tag.